### PR TITLE
Update connector quickstart guides to point to the new Visual Editor

### PR DESCRIPTION
**Pull request by brionmario**
### Purpose

Update connector quickstart guides to point to the new Visual Editor

### Related Issues
- Fixes https://github.com/wso2/product-is/issues/19604

### Related PRs
- N/A

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/connections/resources/guides/apple/quick-start.tsx [hunk 1]
```diff
@@ -109,33 +110,20 @@ const AppleAuthenticatorQuickStart: FunctionComponent<AppleAuthenticatorQuickSta
                             i18nKey={ "extensions:develop.identityProviders.apple.quickStart.steps" +
                             ".selectDefaultConfig.content" }
                         >
-                            Go to <strong>Login Flow</strong> tab and click on <strong>Add Apple login
-                            </strong> to configure a Apple login flow.
+                            Go to <strong>Login Flow</strong> tab and click on the <strong>Add Sign In Option</strong>
+                            button inside the login box. And select a Apple connection.
```
Comment (dasuni-30) [inline apps/console/src/features/connections/resources/guides/apple/quick-start.tsx:114]:
```suggestion
                            button inside the login box. And select an Apple connection.
```

File: apps/console/src/features/connections/resources/guides/swe/quick-start.tsx [hunk 2]
```diff
@@ -210,33 +211,20 @@ const SIWEAuthenticationProviderQuickStart: FunctionComponent<SIWEAuthentication
                                 ".content"
                             }
                         >
-                            Go to <strong>Login Flow</strong> tab and click on <strong>Start with default
-                            configuration</strong>.
+                            Go to <strong>Login Flow</strong> tab and click on the <strong>Add Sign In Option</strong>
+                            button inside the login box. And select a Sign In With Ethereum connection.
```
Comment (dasuni-30) [inline apps/console/src/features/connections/resources/guides/swe/quick-start.tsx:215]:
```suggestion
                            button inside the login box. And select a Ethereum connection.
```

File: apps/console/src/features/connections/resources/guides/swe/quick-start.tsx [hunk 3]
```diff
@@ -210,33 +211,20 @@ const SIWEAuthenticationProviderQuickStart: FunctionComponent<SIWEAuthentication
...
-                    <GenericIcon inline transparent icon={ BuildLoginFlowIllustration } size="huge"/>
+                    <GenericIcon inline transparent icon={ BuildLoginFlowStep01Illustration } size="huge"/>
+                    <GenericIcon inline transparent icon={ BuildLoginFlowStep02Illustration } size="huge"/>
+                    <GenericIcon inline transparent icon={ BuildLoginFlowStep03Illustration } size="huge"/>
                 </>
             ),
             stepTitle: (
                 <Trans i18nKey="extensions:develop.identityProviders.siwe.quickStart.steps.selectDefaultConfig.heading">
-                    Select <strong>Start with default configuration</strong>
+                    Add a <strong>Sign In With Ethereum</strong> connection
```
Comment (dasuni-30) [inline apps/console/src/features/connections/resources/guides/swe/quick-start.tsx:225]: Let's verify this as well

**Comment by dasuni-30** [comment]
Let's add the changeset

File: apps/console/src/features/connections/resources/guides/swe/quick-start.tsx (hunk 2 above)
Comment (brionmario) [reply apps/console/src/features/connections/resources/guides/swe/quick-start.tsx:215]: `Sign In With Ethereum`(https://login.xyz/) is the actual name of the service we are using here. `Ethereum` is the underneath blockchain.

File: apps/console/src/features/connections/resources/guides/swe/quick-start.tsx (hunk 3 above)
Comment (brionmario) [reply apps/console/src/features/connections/resources/guides/swe/quick-start.tsx:225]: Refer: https://github.com/wso2/identity-apps/pull/5551#discussion_r1492035610

**Review by dasuni-30 — APPROVED** [review]

**Pull request merged**